### DISCUSSION TITLE
Italian language

### DISCUSSION THREAD

**Discussion by Magneticdud** [ente-io/ente#3713] [Enhancements]
I saw that in photos web and in photos desktop the Italian language is missing, but when i went to crowdin to translate the strings, the translation is 99% done (it's missing a translation for "generic_error" and seeing the underscore I'd leave it like that)

Does it need something else? Like to approve or proofread? Can I help in some way?

**Comment by mnvr** [comment]
Added!

<img width="371" alt="Screenshot 2024-10-16 at 09 14 22" src="https://github.com/user-attachments/assets/e3f1c980-b590-4cca-bcc3-10d7436e9ab9">

**Reply by Magneticdud** [reply]
Hi I noticed there was a mistake in a string, which prevents users to see terms and privacy.

![Screenshot_20241020-231155](https://github.com/user-attachments/assets/18a048c9-7c1b-4451-bb04-c89be350ff38)

I fixed it right now in crowdin

**Reply by mnvr** [reply]
Thank you! Updated.
